v2/googlecloud-to-elasticsearch/src/test/resources/EventMetadata/inputMessage.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
can you rename this file to inputGCPAuditlogMessage.json so it is clear this is an GCP auditlog?